docs/src/security/data-collection.md

```diff
@@ -2,41 +2,45 @@
 title: "Data collection"
 weight: 3
 description: |
-  As part of our normal business operations we do collect various pieces of data.
+  As part of our normal business operations, we collect various types of data.
 ---
 
 {{< description >}}
 
-In GDPR terms:
+The following list describes the categories of data we collect in GDPR terms:
 
-* Article 4: Our accounts system contains some (routine) Article 4 items (name, address, phone, etc.) in order to allow us to bill your account appropriately.  This information can be verified, changed, and deleted by [logging into your account](https://accounts.platform.sh/).
-* Article 9: We don't capture and store any Article 9 special identifiers (such as race, religion, sexual orientation, or other attributes that are irrelevant to our business).
-* Article 30: The only Article 30 items we keep are IP address and Log files. These reside on AWS/Azure/Orange (depending on your hosting), and may be sent to Sentry.io when there are crashes.
+* Article 4 'personal data': Our accounts system contains some 'personal data' (name, address, phone, etc.) to allow us to bill your account appropriately.  This information can be verified, changed, and deleted by [logging into your account](https://accounts.platform.sh/).
```

Comment:
These should be using double-quotes, not single-quotes.  (For the whole section.)

Reply:
@Crell You're right, thanks! Should I press "Convert to draft" to fix and then re-commit or is there an easier way?